.build/parent-pom-template.xml

```diff
@@ -611,13 +611,13 @@
       <dependency>
         <groupId>org.openjdk.jmh</groupId>
         <artifactId>jmh-core</artifactId>
-        <version>1.21</version>
+        <version>1.36</version>
         <scope>test</scope>
       </dependency>
       <dependency>
         <groupId>org.openjdk.jmh</groupId>
         <artifactId>jmh-generator-annprocess</artifactId>
-        <version>1.21</version>
+        <version>1.36</version>
```

Comment:
nit: Any chance we can move these to 1.37? I'm getting failures pulling the dependency w/ 1.36...